secio/Cargo.toml

```diff
@@ -12,15 +12,15 @@ libp2p-core = { path = "../core" }
 log = "0.4.1"
 protobuf = "2.0.2"
 rand = "0.3.17"
-ring = { version = "0.12.1", features = ["rsa_signing"] }
+ring = { version = "0.13.2", features = ["rsa_signing"] }
```

Comment:
This is unfortunate, but this change will cause tons of problems in the code that's using rust-libp2p right now.